Library/Formula/dirac.rb

```diff
@@ -1,20 +1,39 @@
-require 'formula'
-
 class Dirac < Formula
-  homepage 'http://diracvideo.org/'
-  url 'http://diracvideo.org/download/dirac-research/dirac-1.0.2.tar.gz'
-  sha1 '895aaad832a54b754e58f77c87d38c0c37752b0b'
+  homepage "http://diracvideo.org/"
+  url "http://diracvideo.org/download/dirac-research/dirac-1.0.2.tar.gz"
+  sha256 "816b16f18d235ff8ccd40d95fc5b4fad61ae47583e86607932929d70bf1f00fd"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
   fails_with :llvm do
     build 2334
   end
 
+  patch :DATA
```

Comment:
Yes, sorry, I was under the impression that patches were preferred over inreplaces.